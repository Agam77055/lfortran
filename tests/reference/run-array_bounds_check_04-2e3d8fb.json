{
    "basename": "run-array_bounds_check_04-2e3d8fb",
    "cmd": "lfortran --no-color {infile}",
    "infile": "tests/errors/array_bounds_check_04.f90",
    "infile_hash": "daa672fe4c03715eb861e1221cb21768fb0b297213ccbe906ea2871c",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "run-array_bounds_check_04-2e3d8fb.stderr",
    "stderr_hash": "8a026a9009f4d0fba284c241930bbd9f47a05f7d33c37e10ce20442c",
    "returncode": 1
}