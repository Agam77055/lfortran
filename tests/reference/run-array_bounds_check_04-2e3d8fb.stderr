Runtime error: Array shape mismatch in subroutine 'my'

Tried to match size 4 of dimension 2 of argument number 1, but expected size is 1
